Resources/Locale/en-US/game-ticking/game-presets/preset-revolutionary.ftl

```diff
@@ -14,7 +14,7 @@ head-rev-briefing =
     Use flashes to convert people to your cause.
     Kill all heads to take over the station.
 
-head-rev-break-mindshield = The Mindshield was destroyed!
+head-rev-break-mindshield = The Mindshield was destroyed! Their brain hemorrhages, Gross!.
```

Comment:
the wording on this is kinda undercutting the severity of it so i'd probably just reword it to "The Mindshield exploded violently!" or something like that.